libbpf-tools/profile.c

```diff
@@ -359,6 +375,50 @@ static int print_count(struct key_t *event, __u64 count, int stack_map)
 	return 0;
 }
 
+static int print_count_folded(struct key_t *event, __u64 count, int stack_map)
```

Comment:
Quite some code duplication here between print_count_folded and print_count. If things need to change in the future, we need to change in two different places. I would like to add a bool parameter to print_count to indicate it whether folding is needed or not.

Reply:
Thank you for your feedback.
I updated the patch to remove duplication. Could you let me know what you think?